PathOfBuildingCommunity/PathOfBuilding
# Fix Flesh and Stone + Maim stacking

Fixes #2861.

### Description of the problem being solved:
Flesh and Stone stacks with Maim Support, but **only** when they're linked together. Currently in PoB Maim Support will stack with Flesh and Stone regardless of what skill its linked to.

### Steps taken to verify a working solution:
- Activate Flesh and Stone. Activate Ancestral Warchief.
- Check DPS on Earthquake.
- Enable Maim Support on Ancestral Warchief. DPS on Earthquake should remain unchanged.
- Enable Maim Support on Flesh and Stone. DPS on Earthquake should go up.

### Link to a build that showcases this PR:
https://pastebin.com/9QTmEqmS
